pkg/utils/sriovnet_ops.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
```

Comment:
nit: for consistency also maybe name this `sriovnet_provider.go` or rename `netlink_provider.go` to `netlink_ops.go ` if we end up doing refactoring.